book.html

```diff
@@ -26,7 +28,7 @@ <h2 class="subtitle" id="copyright-c-2017-nicolas-p-rougier-nicolas-rougier-inri
 <img alt="data/nc.large.png" src="data/nc.large.png" style="width: 40px;" />
 <div class="line-block">
 <div class="line"><br /></div>
-<div class="line">Latest version - February 2017</div>
+<div class="line">Latest version - Февраль 2017</div>
```

Comment:
Oh, my default locale is ru-RU. I think in rst2html we need to explicitly specify that we want to use en-EN